cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/relational/RelationalTransform.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.etl.api.relational;
+
+import java.util.List;
+
+/**
+ * An analytical or transformation plugin can implement this interface to provide an alternative
+ * declarative way of doing transformations. If relational transform call is successfull, the transform
+ * will be passed to the underlying engine and plugin will not be called to transform each row.
```

Comment:
does this mean to say `now`?

Reply:
it's `not`. What I mean to say is that in this mode plugin is called to define the transformation, but is not called to transform each separate row